content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -169,7 +169,7 @@ Now that a Key Value instance has been created, we'll add a secret to it and the
 
 Figure 6
 
-### **Authorize the Web App to access your Key Vault**
+### Authorize the Web App to access your Key Vault
 
 To provide access to the secret we created, follow the steps mentioned below:
```

Comment:
Use either `you` or `we` consistently throughout the article. Please don't use both, which would cause inconsistency and may confuse readers.

Reply:
Also, just saying `steps below` would suffice. No need to say `steps mentioned below`.

Reply:
I've made the changes, thanks!

Reply:
> 
> 
> Use either `you` or `we` consistently throughout the article. Please don't use both, which would cause inconsistency and may confuse readers.

I've used "you" throughout this article.